notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -96,8 +96,8 @@
       "source": [
```

Comment:
<p>we -&gt; you</p><p> </p><p>For a complete list of supported feature transformation configs and examples, please go&nbsp;<a href="https://google-cloud-pipeline-components.readthedocs.io/en/google-cloud-pipeline-components-1.0.15/google_cloud_pipeline_components.experimental.automl.tabular.html#google_cloud_pipeline_components.experimental.automl.tabular.FeatureTransformEngineOp" target="_blank">here</a>. -&gt;</p><p> </p><p>Learn more about [supported feature transformation configs and examples](link)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Updated.</p>